lib/mixlib/install/backend/bintray.rb

```diff
@@ -71,6 +73,30 @@ def info
           artifacts.length == 1 ? artifacts.first : artifacts
         end
 
+        #
+        # Makes a GET request to bintray for the given path.
+        #
+        # @param [String] path
+        #   "/api/v1/packages/chef" is prepended to the given path.
+        #
+        # @return [String] JSON parsed string of the bintray response
+        #
+        def bintray_get(path)
+          uri = URI.parse(endpoint)
+          http = Net::HTTP.new(uri.host, uri.port)
+          http.use_ssl = (uri.scheme == "https")
+
+          full_path = File.join(uri.path, "packages/chef", path)
```

Comment:
Would there be any gain to doing `def bintray_get(path, subject = 'chef')` now?  Or is it cleaner to let this be until the `bintray-client` refactor?